src/guild/Player.ts

```diff
@@ -244,12 +249,12 @@ export class Player extends EventEmitter {
 
     /**
      * Move player to another node
-     * @param name Name of node to move to, or the default ideal node
+     * @param opts An object that conforms to the MoveOptions type
      * @returns true if the player was moved, false if not
      */
-    public async move(name?: string): Promise<boolean> {
+    public async move(opts: MoveOptions = { force: false }): Promise<boolean> {
```

Comment:
```suggestion
    public async move(name?: string): Promise<boolean> {
    public async move({ name, force }: MoveOptions): Promise<boolean> {
```
Accept both so API doesn't change